.drone.yml

```diff
@@ -104,8 +112,12 @@ services:
     image: postgres:9.6
     environment:
       - POSTGRES_USER=postgres
+    tmpfs:
+      - /var/lib/postgresql/data
   mysql:
     image: mysql:5.6.27
     environment:
       - MYSQL_DATABASE=test
       - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+    tmpfs:
+      - /var/lib/mysql
```

Comment:
at this time, tmpfs is only available to trusted repositories. Because drone is a public repository and we accept untrusted pull requests, we cannot enable trusted mode.